config/initializers/muli_json.rb

```diff
@@ -0,0 +1 @@
+MultiJson.use :oj
```

Comment:
I considered using Oj for our pubsub system too, but that would have required too many code changes.